# an object that can be grasped in principle
string frame_id
int64 obj_type
geometry_msgs/Pose[] push_poses
float64 dist_to_obj
geometry_msgs/Point[] corner_points
geometry_msgs/Pose mean
float64 width
float64 length
float64 height

